tests/manual-test-cases/Group5-Functional-Tests/5-25-OPS-User-Grant.robot

```diff
@@ -142,24 +142,20 @@ vic-machine create grants ops-user perms
     Cleanup VIC Appliance On Test Server
 
 granted ops-user perms work after upgrade
-    ${status}=  Get State Of Github Issue  7796
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 5-25-OPS-User-Grant.robot needs to be updated now that Issue #7796 has been resolved
-    Log  Issue \#7796 is blocking implementation  WARN
-
-    #Install VIC with version to Test Server  v1.3.0  additional-args=--ops-user ${ops_user_name} --ops-password ${ops_user_password} --ops-grant-perms
-    #
-    #Check Original Version
-    #Upgrade
-    #Check Upgraded Version
-    #
+    Install VIC with version to Test Server  v1.3.1  additional-args=--ops-user ${ops_user_name} --ops-password ${ops_user_password} --ops-grant-perms
+
+    Check Original Version
+    Upgrade
+    Check Upgraded Version
+
     # Run a govc test to check that access is denied on some resources
-    #Attempt To Create Resource Pool
-    #
-    #Run Regression Tests
-    #
-    #Run privilege-dependent docker operations
-    #
-    #Cleanup VIC Appliance On Test Server
+    Attempt To Create Resource Pool
+
+    Run Regression Tests
+
+    Run privilege-dependent docker operations
+
+    Cleanup VIC Appliance On Test Server
```

Comment:
do we want to make `Cleanup VIC Appliance On Test Server` a teardown or run if test failed? I noticed when it failed before cleaning up, no logs were collected because collecting logs is part of the cleanup process

Reply:
Is it not covered at the top by `Test Teardown  Run Keyword If Test Failed  Gather vSphere Logs`

Reply:
that only gathers vSphere logs but not container logs and appliance tether debug log i think

Reply:
Good call, I added `[Teardown]` for all `Cleanup VIC Appliance On Test Server` calls in this suite.